spec/03-plugins/17-jwt/03-access_spec.lua

```diff
@@ -232,13 +238,13 @@ describe("Plugin: jwt (access)", function()
     it("verifies JWT", function()
       PAYLOAD.iss = rsa_jwt_secret_1.key
       local jwt = jwt_encoder.encode(PAYLOAD, fixtures.rs256_private_key, 'RS256')
-      local authorization = "Bearer "..jwt
+      local authorization = "Bearer " .. jwt
```

Comment:
(style updates of legacy - unrelated - code should not be part of such a feature PR, is what I mean)